docs/products/m3db/howto/grafana.rst

```diff
@@ -0,0 +1,37 @@
+Visualise M3DB Data with Grafana
+================================
+
+Since M3DB is best for time series data, many individual metrics, then it's nicer to visualise the data than try to view it in a table or log. Luckily, Aiven can set up the Grafana and the integration between the two services for you.
```

Comment:
"many individual metrics" -> "composed by many individual metrics"

Reply:
I went with "consisting of" but yes, good catch